hw/top_earlgrey/ip/xbar_main/data/autogen/xbar_main.gen.hjson

```diff
@@ -293,6 +301,78 @@
       stub: false
       pipeline: "true"
     }
+    {
+      name: entropy_src
+      type: device
+      clock: clk_main_i
+      reset: rst_main_ni
+      pipeline_byp: "false"
+      inst_type: entropy_src
+      addr_range:
+      [
+        {
+          base_addr: 0x41160000
+          size_byte: 0x1000
+        }
+      ]
+      xbar: false
+      stub: false
+      pipeline: "true"
+    }
+    {
+      name: csrng
+      type: device
+      clock: clk_main_i
+      reset: rst_main_ni
+      pipeline_byp: "false"
+      inst_type: csrng
+      addr_range:
+      [
+        {
+          base_addr: 0x41150000
+          size_byte: 0x1000
+        }
+      ]
+      xbar: false
+      stub: false
+      pipeline: "true"
+    }
+    {
+      name: edn0
+      type: device
+      clock: clk_main_i
+      reset: rst_main_ni
+      pipeline_byp: "false"
+      inst_type: edn
+      addr_range:
+      [
+        {
+          base_addr: 0x41158000
```

Comment:
CC @eunchan

Reply:
The last time I looked at a memory map, there was not enough space open to add these addresses without overlapping an existing block. If this has changed, please provide a pointer to the new map. It may be that the memory map still needs changing, but not sure what the process is to do that.